design/account.go

```diff
@@ -206,3 +245,20 @@ var identityDataAttributes = a.Type("IdentityDataAttributes", func() {
 	a.Attribute("username", d.String, "The username")
 	a.Attribute("providerType", d.String, "The IDP provided this identity")
 })
+
+var createUserDataAttributes = a.Type("CreateIdentityDataAttributes", func() {
+	a.Attribute("userID", d.String, "The id of the corresponding User")
```

Comment:
This PR should re-use https://github.com/fabric8-services/fabric8-auth/pull/164 and also add a cluster info to the payload/DB.